<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="methods" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="seleniumcourse.testcomponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="methods" name="LoginTest(failed)">
    <classes>
      <class name="seleniumcourse.tests.LoginErrormessageTest">
        <methods>
          <include name="validCred"/>
          <include name="closeBrowser"/>
          <include name="wrongPassword"/>
          <include name="launchApplication"/>
          <include name="wrongUsername"/>
        </methods>
      </class> <!-- seleniumcourse.tests.LoginErrormessageTest -->
    </classes>
  </test> <!-- LoginTest(failed) -->
  <test thread-count="5" parallel="methods" name="SubmitOrderTest(failed)">
    <classes>
      <class name="seleniumcourse.tests.Standalonetest_PO">
        <methods>
          <include name="closeBrowser"/>
          <include name="standaloneTest" invocation-numbers="0"/>
          <include name="launchApplication"/>
          <include name="confirnOrder"/>
        </methods>
      </class> <!-- seleniumcourse.tests.Standalonetest_PO -->
    </classes>
  </test> <!-- SubmitOrderTest(failed) -->
</suite> <!-- Failed suite [Suite] -->
